confobo/command/decorators.py

```diff
@@ -18,4 +18,4 @@ def wrapper(*args, **kwargs):
         r = c(*args, **kwargs)
         return r
 
-    return wrapper
\ No newline at end of file
+    return wrapper
```

Comment:
_where is my "ku klux klan" picture with the "always put newline at the end of file" text?_